xapian-applications/omega/worker.cc

```diff
@@ -25,15 +25,16 @@
 #include "worker.h"
 #include "worker_comms.h"
 
-#include "pkglibbindir.h"
 #include <csignal>
 #include <cstring>
 #include <cerrno>
-#include <iostream>
-#include "safefcntl.h"
-#include "safeunistd.h"
```

Comment:
Each `"safeXXX.h"` header is essentially a compatibility stand-in for the respective `<XXX.h>` system header, which is why they were in this block of system headers.  That's the convention that's generally used in the code base, and I think it makes sense - it means that someone going to add (say) `<fcntl.h>` is more likely to notice that "safefcntl.h" is already there, and it means we don't have to moved header includes around in a lot of files if a new "safeXXX.h" is introduced, or an existing one eliminated (which would happen if the compatibility handling it was doing is no longer needed for any current platform).